apis/vshn/v1/vshn_nextcloud.go

```diff
@@ -174,6 +177,13 @@ func (v *VSHNNextcloud) SetInstanceNamespaceStatus() {
 	v.Status.InstanceNamespace = v.GetInstanceNamespace()
 }
 
```

Comment:
Nitpick: I would add a comment as we do for other types

Reply:
I'd say it's not a nitpick. These comments will be shown to the users when they use `kubectl explain` so they are part of the end-user docs.

Also, they are used for he automatically generated reference configs on https://docs.appcat.ch/

Reply:
Ah indeed, I knew it's important to have these comments but forgot that they end up in the CR definition. So yeah we need every comment we can get.